package.json

```diff
@@ -20,6 +21,8 @@
   "homepage": "https://github.com/labs15-best-places/backend#readme",
   "dependencies": {
     "bcryptjs": "^2.4.3",
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
```

Comment:
Possible that we need to keep test packages in Dev dependencies.
But it's shouldn't be critical, right?...

Reply:
I think it doesn't matter